cnxarchive/views.py

```diff
@@ -368,6 +368,16 @@ def html_date(datetime):
     return datetime.strftime("%a, %d %b %Y %X %Z")
 
 
+def html_rss_date(datetime):
+    """
+    Return the HTTP-date format of python's datetime time.
+
+    Based on:
+    https://legacy.cnx.org/content/recent.rss
+    """
+    return datetime.strftime("%b %d, %Y at %I:%M %p")
```

Comment:
There's a spec that says this is supposed to be RFC822 (or perhaps RFC2822) formatted. The correct call here would be `strftime("%a, %d %b %Y %H:%M:%S %z")`  (I note that the legacy feed also gets this wrong, but not the same way this code does :-) )